styles/index.css

```
@@ -1,55 +1,194 @@
+.editLink:hover {
+  color: var(--rp-c-brand-dark);
+}
+
+.paginationItem {
+  font-weight: 500;
+  cursor: pointer;
+  font-size: var(--text-sm);
+  line-height: var(--tw-leading, var(--text-sm--line-height));
+
+  --tw-font-weight: var(--font-weight-medium) /* 500 */;
+  font-weight: var(--font-weight-medium) /* 500 */;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-weight is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)